ui/app/templates/components/keymgmt/key-edit.hbs

```diff
@@ -168,7 +168,7 @@
     {{/each}}
   {{else}}
     <div class="has-top-margin-xl has-bottom-margin-s">
-      <h2 class="title is-5 has-border-bottom-light">Key details</h2>
+      <h2 class="title is-5 has-border-bottom-light has-bottom-padding-s">Key details</h2>
```

Comment:
Noticed a padding issue. Did a quick fix, though it's a little strange because the h2 here is the same as the h2 below and there is no issue there. Something to do with the margin not being respected by the InfoTableRow. Didn't notice this issue anywhere else with similar setups so settled on a helper.

Before 
<img width="401" alt="image" src="https://github.com/hashicorp/vault/assets/6618863/4744b16f-12f1-4013-ab76-9b8e4a8fb358">

After
<img width="337" alt="image" src="https://github.com/hashicorp/vault/assets/6618863/2ea80cea-e80e-4a45-9f41-e163a41e17cb">